src/Charts/BentoPie.tsx

```diff
@@ -119,16 +117,30 @@ const BentoPie = ({
   );
 };
 
-const RenderLabel: PieProps['label'] = (params) => {
-  const { cx, cy, midAngle, outerRadius, fill, payload, index, activeIndex } = params;
+const getPropNumber = (val: number | string | undefined, defaultValue?: number): number => {
+  let value = defaultValue || 0
+  if (val && typeof val === "string") {
+    value = Number(val)
```

Comment:
other parts of the codebase use semicolons; this should be enforced via lint and (imo) lines should preferably be semicolon-terminated